### DISCUSSION TITLE
Testimonials for the new website

### DISCUSSION THREAD

**Discussion by Griefed** [Griefed/ServerPackCreator#443] [Q&A]
Hello everyone!

I'm reworking the website for ServerPackCreator and adding a section with testimonials from modpack-makers about how ServerPackCreator has improved their modpack-making-experience.
If you're using ServerPackCreator for your modpacks and if it has improved your experience with maintaining your modpack, you can submit a testimonial in this discussion, which will then be added to the testimonials on the new website.

A testimonial will look like the following:
![testimonials-empty](https://user-images.githubusercontent.com/44273438/204213043-0634f76f-3786-4a8c-bae7-a36a80e709f5.png)
For live action examples, see https://serverpackcreator.de and scroll to the bottom. 😊

In order to have yours added, I require the following:

1. Your name as it is displayed on CurseForge
2. Your modpacks icon
2. Your modpacks name
3. The quote, maximum 350 characters

As for the quote itself: Apart from the already mentioned character limit, it should be related to ServerPackCreator and how/where it helps/helped you work and release new versions of your modpack. If applicable, of course 😄

If you are part of a team which works on a modpack together, please talk the other team-members first before submitting a testimonial.

Thanks in advance.
Cheers,
Griefed

**Comment by kreezxil** [comment]
**Name**: Kreezxil
**Icon**: 
![637379312073959046](https://user-images.githubusercontent.com/3880072/204245446-3e4b642f-b345-4a15-b3b7-2c746d30a161.png)

**PackName**: World of Dragons
**Quote**:
> Before ServerPackCreator I would use my own well-documented manual method to create server packs. See https://kreezcraft.com/uupt and also https://kreezcraft.com/n0cp. The two methods are very similar but time consuming. ServerPackCreator (SPC) allows you to do all of that with a few simple quick button clicks! Because of this, I'm also a sponsor.

**Reply by Griefed** [reply]
Done. Thank you so much! 😁
![image](https://user-images.githubusercontent.com/44273438/204379673-cc507e11-106a-49d7-899d-3ffb76340e84.png)

**Comment by thrasos-dev** [comment]
**Name**: Kyber_6
**Icon**:
<picture>![638032053304092606](https://user-images.githubusercontent.com/100732009/204262515-bd27ebbb-beff-4b74-b68a-a5298ac22f81.png)</picture>
**Modpack Name**: Fantasy Minecraft FABRIC
**Quote**: "ServerPackCreator has made the process of creating Server Packs for our Modpack much more quick and efficient."

**Reply by Griefed** [reply]
Done. Thank you very much! 😁
![image](https://user-images.githubusercontent.com/44273438/204379794-131906ee-4156-43f5-830f-0d731a30c291.png)

**Comment by DKLYNTLY** [comment]
**Name:** Darkly
**Icon:** 
![image](https://user-images.githubusercontent.com/75499681/204279943-9d4f7222-5784-4090-9943-3b5ae23ed6e7.png)
**modpacks name:** Modded Oneblock Skyblock 
**quote:** Server pack creator is like a Instant Pot, it has everything you need to make a yummy server pack

**Reply by Griefed** [reply]
Done. Thank you very much for your contribution! 😁
![image](https://user-images.githubusercontent.com/44273438/204380216-3c289819-ca7e-4a17-8294-34e14c191b0c.png)

**Reply by DKLYNTLY** [reply]
yw i enjoyed coming up with the quote

**Comment by ModernGamingWorld** [comment]
![BH_Monumental_Experience_Logo](https://github.com/Griefed/ServerPackCreator/assets/75923060/50236ad6-9a94-46a4-bf80-9bed44a87026)
Modpack Name: Monumental Experience
Quote: "ServerPackCreator has made the process of creating Server Packs for our Modpack much more quick and efficient."

**Reply by Griefed** [reply]
Thanks for the submission!
Are you sure you would like that to be the quote to display on the ServerPackCreator website? It's the same as Kyber_6's one for Fantasy Minecraft. :D

**Comment by Griefed** [comment]
Further submissions can be submitted via the [Testimonial Submission Template](https://github.com/Griefed/ServerPackCreator/issues/new?assignees=Griefed&labels=&projects=&template=testimonial.yml&title=%5BTestimonial+Submission%5D%3A+%3CName+of+your+Modpack%3E)
